exporter/azuremonitorexporter/azuremonitor.go

```diff
@@ -18,18 +18,35 @@ type azuermonitorconfig struct {
 
 // AzureMonitorExportersFromViper unmarshals the viper and returns exporter.TraceExporters targeting
 // Azure Monitor according to the configuration settings.
-func AzureMonitorExportersFromViper(v *viper.Viper) (tps []consumer.TraceConsumer, mps []consumer.MetricsConsumer, doneFns []func() error, err error) {
+func AzureMonitorExportersFromViper(v *viper.Viper) (traceExporters []consumer.TraceConsumer, mps []consumer.MetricsConsumer, doneFns []func() error, err error) {
```

Comment:
what is mps?

Reply:
metricsExporter - I will rename this so this is clear if someone adds one in the future.